src/chevah_compat/posix_filesystem.py

```diff
@@ -763,6 +756,11 @@ def _dirEntryToFileAttributes(self, entry):
             if path.startswith('\\\\?\\') and path[5] == ':':
                 path = path[4:]
 
+        hardlinks = stats.st_nlink
+        if not hardlinks and os.name == 'nt':
+            # I don't know why on Windows we doing scandir.
```

Comment:
The comment was bad... the comment was about scandir returning `hardlinks=0`

we use scandir for speed and most importantly not to block the process.

With os.listdir, if you have 100.000 files in a dir, when you call it it will return all of them in one go, possible blocking the process.

with scandir we have an iterator